src/requests/user.js

```diff
@@ -92,17 +92,11 @@ const details = function (token = '', userId = '') {
 const list = function (userType, pageNo, pageSize, searchText, organization_id = null, body = {}) {
 	return new Promise(async (resolve, reject) => {
 		try {
-			let apiUrl =
-				userBaseUrl +
-				endpoints.USERS_LIST +
-				'?type=' +
-				userType +
-				'&page=' +
-				pageNo +
-				'&limit=' +
-				pageSize +
-				'&search=' +
-				searchText
+			let apiUrl = userBaseUrl + endpoints.USERS_LIST + '?type=' + userType
```

Comment:
@adithyadinesh0412 any issue with above url constructing method

Reply:
Above implementation is expecting page and limit. So added logic to check and attach the query params. @priyanka-TL

Reply:
optimize this code